plugins/modules/cloud/misc/proxmox_template.py

```diff
@@ -168,7 +178,7 @@ def main():
             api_password=dict(no_log=True),
             validate_certs=dict(type='bool', default=False),
             node=dict(),
-            src=dict(type='path'),
+            src=dict(type='path', aliases=['path']),
```

Comment:
I'm in favor of not adding this alias (and remove its mention from the docs). I personally would avoid aliases nowadays, except in situation where they are needed for backwards compatibility or when they really make sense. I don't think in this situation this alias is needed.

Are there other opinions?

Reply:
I removed the `path` alias from doc, it didn't work until now and most probably created confusion for people writing roles based on doc.